juniper_official/Solutions/System-blackhole-detection/system-traffic.rule

```diff
@@ -0,0 +1,128 @@
+/*
+ * Checks system total input and output packets count periodically and 
+ * notifies anomaly when total input and output packets count is abnormal.
+ * Two dynamic inputs control detection
+ * 
+ *   1) traffic-diff-threshold, is the threshold that causes the rule to
+ *      report a anomaly. This rule will set a dashboard color to red when
+ *      system input($pfe-input-packets) and output ($pfe-output-packets)
+ *      traffic difference exceeds static threshold 'traffic-diff-threshold'.
+ * 
+ */
+iceberg {
+    topic system.traffic {   
+        description "This topic is to monitor system total input and output packets usage and notify in case difference in io traffic is above static and dynamic threshold";
+        synopsis "System total Input and output packets analyzer";
+        rule check-system-traffic {
+            synopsis "system traffic statistics analyzer";
+            description "Collects system total io packets and notifies in case of anomalies when io traffic difference is above static and dynamic threshold";
+            /*
+             * Monitors the system total input and output packets periodically. Notifies via
```

Comment:
Can we please add the host path traffic and the discard traffic as well?

Reply:
I have in person discussion with kernel and pfe folks. Currently there is no straight way to collect host path traffic. But I will find some way and incorporate post HB 2.0.0